src/server.ts

```diff
@@ -1 +1,13 @@
-console.log('Hello server !!!!!!');
+import dotenv from 'dotenv';
+dotenv.config();
+import express from 'express';
+
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.get('/', (_reg, res) => {
```

Comment:
```suggestion
app.get('/', (_req, res) => {
```
reguest 😂 sounds like a german accent